content/configuration/_index.md

```diff
@@ -51,3 +51,11 @@ Or:
 ```
 oc exec -n openshift-sdn $(oc get pods -n openshift-sdn -l app=sdn --no-headers=true -o custom-columns=:metadata.name|head -n1) cat /config/{kube-proxy-config,sdn-config}.yaml
 ```
+
+# Making Master Unscheduleable/Scheduleable
+To configure master as unscheduleable (when UPI installation without setting this prior to the install):
+```
+#oc patch --type=merge --patch='{"spec":{"mastersSchedulable": false}}' schedulers.config.openshift.io cluster 
```

Comment:
Can you remove the `#` to be consistent with the rest of the tips? Thanks!

Reply:
fixed this as well.